src/Homework6.py

```diff
@@ -0,0 +1,124 @@
+"""Create simulator from real life. This can be booking room in hotel,
+visit to casino, visit to bar. Create 3-4 objects, that can
+describe situation.
+Objects should contain attributes and methods to simulate some use cases.
+Completed program should print object states,
+it actions (methods) and objects interaction.
+"""
+
+
+class Bar(object):
+    def __init__(self, name, vodka_price, beer_price, wine_price):
+        self.name = name
+        self.vodka_price = vodka_price
+        self.beer_price = beer_price
+        self.wine_price = wine_price
+
+    def menu(self):
+        print('Бар', self.name)
+        print('Водка', self.vodka_price)
+        print('Пиво', self.beer_price)
+        print('Вино', self.wine_price)
+
+
+class ManInBar(object):
+    def __init__(self, name):
+        self.name = name
+        self.money = 100
+        self.ethanol = 0
+        self.danger_level = 'OK'
+        self.card_limit = 1
+
+    def drink(self, type_of_drink, place):
+        if self.danger_level == 'DROP DEAD':
+            print('No more tonight, wait until tomorrow')
+        else:
+            if type_of_drink == 'vodka':
+                if self.money - place.vodka_price >= 0:
+                    self.money -= place.vodka_price
+                    self.ethanol += 0.04
+                else:
+                    print('Not enough funds')
+            elif type_of_drink == 'beer':
+                if self.money - place.beer_price >= 0:
+                    self.money -= place.beer_price
+                    self.ethanol += 0.025
+                else:
+                    print('Not enough funds')
+            elif type_of_drink == 'wine':
+                if self.money - place.wine_price >= 0:
+                    self.money -= place.wine_price
+                    self.ethanol += 0.03
+                else:
+                    print('Not enough funds')
+            else:
+                print('No such drinks in our place')
+        if self.ethanol >= 0.15:
+            self.danger_level = 'DROP DEAD'
+        elif self.ethanol >= 0.07:
+            self.danger_level = 'DRUNK'
+
+    def info(self):
+        print('И вот перед нами', self.name)
+        print('У него в кармане', self.money)
+        print('Уровень этанола в крови', round(self.ethanol, 3))
+        print('Состояние -', self.danger_level)
```

Comment:
as everywhere you use russian it seems it should be mapper from english danger_level to russian one